src/app/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import { Header, MovieList } from './Components';
+
+const App = () => (
+  <>
+    <Header />
```

Comment:
The Header styling needs some responsive adaptation. Maybe wrapping the items, or displaying them as a column